owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileActivity.java

```diff
@@ -520,4 +491,30 @@ public void onFailedSavingCertificate() {
     public void onCancelCertificate() {
         // nothing to do
     }
+
+    /**
+     * Implements callback methods for service binding. Passed as a parameter to {
```

Comment:
Passed as a parameter to what?

Reply:
🤔  Im not sure. It was there. It was reordered by 'Reformat file'

Reply:
It was there before, it was reordered by 'Reformat file' 🤔